addons/account/tests/test_account_move_out_invoice.py

```diff
@@ -2630,6 +2630,29 @@ def test_out_invoice_switch_out_refund_2(self):
             'amount_untaxed' : self.move_vals['amount_untaxed'],
         })
 
+    def test_out_invoice_switch_out_refund_3(self):
+        # Test creating an account_move with an out_invoice_type and switch it in an out_refund and a fiscal position.
```

Comment:
Not very clear, I would go with something like
```suggestion
        """Test that switching out_invoice_type to out_refund keep the fiscal position."""
```